### DISCUSSION TITLE
slow loading, degraded performance

### DISCUSSION THREAD

**Discussion by uncomplexity** [community/community#137098] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Bug

### Body

just getting stuck on endless loading

noticed this at three sites already

- kysely docs
- tree sitter docs
- our own project's docs

![image](https://github.com/user-attachments/assets/009cc7ba-3f1c-467d-acdc-c8e288277203)

**Comment by theshawnv** [comment, marked as answer]
I tested those sites you mentioned, and I didn't see any problem, I think the problem is your local machine, check your DNS and make sure you don't set any proxy on your local machine.

**Comment by Yackens** [comment]
I checked them as well and they seem to be working as expected. My advice is that you revise your internet connection and your computer to make sure that everything is working as expected.

If you found this answer useful, please don't forget to mark it as the accepted answer so more people can see it.

**Comment by Keller896** [comment]
If you're experiencing slow loading and degraded performance on your system, it could be due to high memory usage. I recommend trying Mem Reduct, a lightweight tool that helps free up RAM and improve overall system performance. It can optimize your memory usage and keep your system running smoothly, especially when multitasking or running heavy applications. Visit now for more information about Mem Reduct: memoryreduct.com
